docs/en/operations/named-collections.md

```diff
@@ -70,6 +69,27 @@ $ cat /etc/clickhouse-server/config.d/named_collections.xml
 </clickhouse>
 ```
 
+## Modifying named collections
+
+Named collections that are created with DDL queries can be altered or dropped with DDL.  Named collections created with XML files can be managed by editing or deleting the corresponding XML.
```

Comment:
extra space before second sentence

Reply:
:) Back in the dark ages, two spaces between sentences were required.